src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncTests.cs

```diff
@@ -501,7 +501,7 @@ .locals init (int V_0,
                       IL_001a:  pop
                       IL_001b:  ldsfld     "Test.<>c Test.<>c.<>9"
                       IL_0020:  ldftn      "void Test.<>c.<F>b__1_0()"
-                      IL_0026:  newobj     "System.Action..ctor(object, nint)"
+                      IL_0026:  newobj     "System.Action..ctor(object, System.IntPtr)"
```

Comment:
https://github.com/dotnet/roslyn/pull/78246 updated our baselines to not use `nint`.